core/src/main/java/org/apache/accumulo/core/clientImpl/TableOperationsImpl.java

```diff
@@ -1450,6 +1452,34 @@ public Text getMaxRow(String tableName, Authorizations auths, Text startRow,
     return finalUsages;
   }
 
+  public static Path findExportFile(ClientContext context, String[] importDirs)
```

Comment:
add a function comment as to what this is doing

Reply:
Added function comment.